idowuadeleke/BANKA
# #165424128 Connect user signup to database

### What does this PR do?
Persist user sign up data to a database

### Description of Task to be completed?
- Write unit test cases for user sign up connected to database

- create API endpoint that takes user signup input and stores them in a database

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ft-165424128-User-Signup-Database`

- run `npm install`

- run `npm run start`

- make a request to `POST localhost:3000/api/v2/auth/signup`
  NB see screenshot below on the request data to pass

### Any background context you want to provide?
NA
### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165424128

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56306564-ba1d7180-613a-11e9-834a-557279964fa1.png)
